database/migrations/2022-06-07-000000_drop_action_receipt_actions_args_amount_index/down.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX action_receipt_actions_args_amount_idx ON action_receipt_actions((args->'args_json'->>'amount'))
```

Comment:
Not exactly sure why these migrations existed in the lake fork and not here, should these still exist?

Reply:
It might have been deleted in the original Indexer for Explorer while Lake fork existed. @telezhnaya do you know something about it?

Reply:
Oh I know now - I somehow missed that the `migrations/` folder still existed in the root repository with these 3 migrations. That's why this was seen as an `git add` rather than a move. I've deleted the root `migrations/` and now it sees that they've been moved.